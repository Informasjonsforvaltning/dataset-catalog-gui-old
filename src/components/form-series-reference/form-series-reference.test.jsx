import React from 'react';
import { shallow } from 'enzyme';
import FormSeriesReference from './form-series-reference.component';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {
    initialValues: {}
  };
  wrapper = shallow(<FormSeriesReference {...defaultProps} />);
});

test('should render FormSeriesReference correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
