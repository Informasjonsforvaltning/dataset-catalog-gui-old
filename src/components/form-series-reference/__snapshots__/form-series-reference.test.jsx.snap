// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormSeriesReference correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
